Source/WebCore/css/parser/CSSSupportsParser.h

```diff
@@ -30,6 +30,7 @@
 #pragma once
 
 #include "CSSParserToken.h"
+#include "CSSSelectorParser.h"
```

Comment:
You might consider making a CSSSelectorEnum header so we don't have to include the whole CSSSelectorParser (in line with @jernoble 's build improvement work!)

Reply:
Sometimes you can just do a forward declaration and not need to create a new header.